actions/plugins/publish/publish/publish.sh

```diff
@@ -42,6 +46,12 @@ if [ -z $gcom_env ]; then
     exit 1
 fi
 
+if [ -z $scopes ]; then
```

Comment:
if scopes is not provided, should we set universal and continue on? It will be always set [here](https://github.com/grafana/plugin-ci-workflows/pull/10/files#diff-ea3ea8c9932adc7ba8161ceda844fedd43b006848ef1140c050cbd7ea0788a18R29), right?

Reply:
Yes, I was not sure about how to handle it. I have changed it so it's `universal` by default as I think it makes more sense now :+1: